drracket/gui-debugger/debug-tool.rkt

```diff
@@ -153,19 +147,20 @@
                              (truncate-value (vector-ref v i) size (sub1 depth)))))]
         [else v]))
     
-    (define filename->defs
-      (lambda (source [default #f])
-        (let/ec k
-          (cond
-            [(is-a? source editor<%>) source]
-            [else
-             (send (group:get-the-frame-group) for-each-frame
-                   (lambda (frame)
-                     (and (is-a? frame drscheme:unit:frame<%>)
-                          (let* ([defss (map (lambda (t) (send t get-defs)) (send frame get-tabs))]
-                                 [defs (findf (lambda (d) (send d port-name-matches? source)) defss)])
-                            (and defs (k defs))))))
-             default]))))
+    (define (filename->defs source [default #f])
+      (let/ec
+       k
```

Comment:
@sorawee this is an unfortunate newline, I'd say. Better if `let/ec` always put the identifier on the same line and then to inside the `let/ec` isntead of to the start of the identifier. Is that possible?

Reply:
I agree. Currently the `let/ec` call is treated like a function application. It would be easy to instruct that it's a special form, which needs to be formatted differently.